### ISSUE TITLE
TXT file contains incorrect spawn locations

### ISSUE THREAD

**Issue by 18swenskiq** [18swenskiq/CS-GO-Auto-Radar#3]
I think this should be a simple fix as an axis seems flipped or something

**Comment by 18swenskiq** [comment]
According to slimek, all the X values are inverted

**Comment by screedlerschoice** [comment]
I observed this behavior of the incorrect spawn location coordinates (for display while the map loads overlaid over the spawn on the radar is that correct?) but in my scenario rather than an inverted X value, the X value was off by a little bit. 

Example data: "CTSpawn_x" **"0.540000"** vs "CTSpawn_x" "0.440000" 
& 
"TSpawn_x" "0.640000" vs "TSpawn_x" **"0.420000"** 
where the bolded value was the correct one after manual adjustment and testing. Happy to submit a vmf for testing consideration if you please just holler!

**Comment by 18swenskiq** [comment]
> I observed this behavior of the incorrect spawn location coordinates (for display while the map loads overlaid over the spawn on the radar is that correct?) but in my scenario rather than an inverted X value, the X value was off by a little bit. 
> 
> Example data: "CTSpawn_x" **"0.540000"** vs "CTSpawn_x" "0.440000" 
> & 
> "TSpawn_x" "0.640000" vs "TSpawn_x" **"0.420000"** 
> where the bolded value was the correct one after manual adjustment and testing. Happy to submit a vmf for testing consideration if you please just holler!

Noted, will keep it in mind. I've actually kinda shifted plans recently towards making my own radar generator from scratch so these things won't be fixed from me for a while. Although if someone else submits a pull request I'll gladly add the fixed to the repo and release a new version, but as of now it isn't my focus

**Comment by screedlerschoice** [comment]
Fair enough! I'm a C# guy so reading this c++ is giving me a good reason to expand my skillset: I will be working towards trying to understand this defect as is. If you need any test data vmf for your rewrite with tar map divider use cases in mind I'd love to help! :)
